dapps/leavingTraces.md

```diff
@@ -18,23 +18,23 @@ As part of our ongoing commitment to transparency Traces, formerly known as Mile
 
-**4.** From this point on, consider copying all of your Trace text to a place where you keep your own records. If your reviewer declines to review this milestone, or if it gets caught in the process for any reason, you will have to re-propose the milestone anew.
+**4.** From this point on, consider copying all of your Trace text to a place where you keep your own records. If your reviewer declines to review this Trace, or if it gets caught in the process for any reason, you will have to re-propose the Trace anew.
 
-**5.** In the "Description" text field below add more details. Include proof of work or expenses. You can link to github pages, other documents you worked on, or upload photos of receipts. Show your calculations if any.
+**5.** In the "Description" text field below add more details. Include proof of work or expenses. You can link to Github pages, other documents you worked on, or upload photos of receipts. Show your calculations if any.
 
 
-**6.** Add a photo for your milestone. It can be related to the work, or not!
+**6.** Add a photo for your Trace. It can be related to your contribution, or not!
 
 
 <img id="contentimg"  alt='Trace Options' src={useBaseUrl('img/content/trace/tracePictureSelect.png')} />
 
-**7.** From **Payment Options** select how you wish to be compensated. Choose from the cryptocurrency options available and provide an address for which you wish to be sent payment. You can also be awesome and give 3% of your amount received to Giveth to keep the platform running(optional)!
+**7.** From **Payment Options** select how you wish to be compensated. Choose from the cryptocurrency options available, and provide an address for which you wish to be sent payment. You can also be awesome and give 3% of your amount received to Giveth to keep the platform running (optional)!
 
-**8.** If at step #2 you chose a *Bounty* or a *Milestone* you have the option to set a Reviewer. Think about who will be the best for the job. The Reviewer is responsible for ensuring that all steps in the Trace have been completed and accomplished before the compensation should be paid out. If possible, choose a different person than the campaign manager. Having the campaign manager, reviewer, and proposer (you) be 3 different people helps ensure against mistakes being overlooked, but the most important thing is that the people approving your Trace understand the work.
+**8.** If at step #2 you choose a *Bounty* or a *Milestone* you have the option to set a Reviewer. Think about who will be the best for the job. The Reviewer is responsible for ensuring that all steps in the Trace have been completed and accomplished before the compensation should be paid out. If possible, choose a different person than the Campaign Manager. Having the Campaign Manager, Reviewer, and Proposer (you) be 3 different people helps ensure against mistakes being overlooked, but the most important thing is that the people approving your Trace understand the work.
```

Comment:
this is referring to a decision made in  a previous step - should be past tense `chose`

Reply:
Actually, because it is hypothetical and instructions, it should be present.